setup.py

```diff
@@ -16,7 +16,7 @@
     name='gimel',
     version=open(os.path.join('gimel', 'VERSION')).read().strip(),
     description='Run your own A/B testing backend on AWS Lambda',
-    long_description=open('README.md').read(),
+    long_description=open('README.rst').read(),
```

Comment:
rst feels kinda ugly compared to markdown though... How about something like [this](http://stackoverflow.com/a/23265673/305019) instead? (keeping the markdown file, but converting it when pushing to pypi)